### ISSUE TITLE
No error on class not found

### ISSUE THREAD

**Issue by nmsobri** [mikecao/flight#69]
Take from the doc:

```
class Greeting {
    public static function hello() {
        echo 'hello world!';
    }
}

Flight::route('/', array('Greetings','hello')); #notice the 's' at the end
```

Flight will output nothing, should have throw error/exception class not found..

**Comment by nmsobri** [comment]
Possible solution 

```
    #Loader::loadClass
    public static function loadClass($class) {
        $class_file = str_replace('\\', '/', str_replace('_', '/', $class)).'.php';

        foreach (self::$dirs as $dir) {
            $file = $dir.'/'.$class_file;
            if (file_exists($file)) {
                require $file;
                return;
            }
        }
        trigger_error(sprintf('Class %s not found', $class), E_USER_ERROR); #handle class not found
        # throw new \ErrorException(sprintf('Class %s not found', $class), E_USER_ERROR);
    }
```

**Comment by mikecao** [comment]
Flight intentionally does not throw an error when it can't find a class. This is because PHP `class_exists` function tries to autoload class by default and throwing an error causes issues with third party libraries. This seems like more of an issue with the router or dispatcher. I'll take a look.

**Comment by mikecao** [comment]
Now fixed in commit 6696c0d267c8e05e3c497588041b5169d057b401. It should now throw an exception for invalid callbacks.
